CHANGES.rst

```diff
@@ -143,6 +143,9 @@ astropy.io.misc
 astropy.io.fits
 ^^^^^^^^^^^^^^^
 
+- ``diff_values()`, ``report_diff_values()``, and ``where_not_allclose()``
+  utility functions are moved to ``astropy.utils.diff``. [#7444]
```

Comment:
if these were not part of the public api before, probably you don't even need this part of the entry, just the addition below.

Reply:
What are the chances of people using them anyway after finding them in the code like I did? :thinking:

Reply:
But good point... I'll remove tomorrow unless I hear otherwise. Thanks!